### DISCUSSION TITLE
Accurate indicator of dynamic length

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#23748] [Ideas] [answered]
osulazer precision indicator has a fixed length, which I don't like. It cannot directly reflect the tolerance. I hope to have a dynamic length precision indicator.

allow precise indicators to be set in the skin editor, and can choose whether to follow the length of the speed change. (DT HT)

**Comment by bdach** [comment]
i have no idea what this means

**Reply by Theighlin** [reply]
I _think_ it's this again https://github.com/ppy/osu/issues/8929
There probably is a more recent thread with a new perspective on this, but it's the same issue.

**Reply by nana123126** [reply]
> _我认为_又是这个#8929 可能有一个更新的线程对此有了新的视角，但这是同一个问题。

I mean that BarHitErrorMeter can adjust its length based on OD difficulty.
Maybe I made a mistake because I used machine translation.

**Reply by bdach** [reply, marked as answer]
Ok so you basically want the bar meter to be narrow with high OD, and wide with low OD?

I guess that is something that *could* be implemented although I'm not sure I see the point myself. Will make sizing the element much more difficult skinning-wise.
